go-controller/pkg/node/management-port-dpu.go

```diff
@@ -84,6 +90,13 @@ func (mp *managementPortRepresentor) Create(_ *routemanager.Controller, nodeAnno
 	if mp.repName != k8sMgmtIntfName {
 		ovsArgs = append(ovsArgs, "external-ids:ovn-orig-mgmt-port-rep-name="+mp.repName)
 	}
+
+	if br_type == types.DatapathUserspace {
+		dpdkArgs := []string{"type=dpdk"}
+		ovsArgs = append(ovsArgs, dpdkArgs...)
+		ovsArgs = append(ovsArgs, fmt.Sprintf("mtu_request=%v", config.Default.MTU))
```

Comment:
why mtu_request is needed?

Reply:
Setting the MTU via netlink changes only the kernel netdevice. This ensures that it is changed on the OVS side as well.

Reply:
ack